src/main/java/logisticspipes/gui/hud/HUDPowerLevel.java

```diff
@@ -60,10 +60,10 @@ public void drawTexturedModalRect(int par1, int par2, int par3, int par4, int pa
         float var8 = 0.00390625F;
         Tessellator var9 = Tessellator.instance;
         var9.startDrawingQuads();
-        var9.addVertexWithUV(par1 + 0, par2 + par6, 0, (par3 + 0) * var7, (par4 + par6) * var8);
```

Comment:
+ 0 in guis means that this is the start point, allows to understand code more quick, so it should remain the same

Reply:
personally I think it makes the code look terrible, but sure